packages/ketcher-react/src/script/ui/dialog/toolbox/sgroup.module.less

```diff
@@ -94,11 +97,9 @@
       width: 100%;
       display: block;
       margin-bottom: 0;
-      margin-top: 4px;
     }
 
     textarea {
-      margin-top: 4px;
       max-width: 234px;
     }
```

Comment:
It may be worth connecting HTML elements together using "&" and ">" so that styles are not overridden, just in case

Reply:
fixed